code/datums/entities/player_sticky_ban.dm

```diff
@@ -131,3 +131,88 @@ BSQL_PROTECT_DATUM(/datum/entity/stickyban)
 		"ip",
 		"linked_stickyban",
 	)
+
+/client/proc/add_known_alt()
+	set name = "Add Known Alt"
+	set category = "Admin.Alt"
+
+	var/player_ckey = ckey(tgui_input_text(src, "What is the player's primary Ckey?", "Player Ckey"))
+	if(!player_ckey)
+		return
+
+	var/datum/entity/player/player = get_player_from_key(player_ckey)
+	if(!istype(player))
+		return
+
+	var/whitelist_to_add = ckey(tgui_input_text(src, "What is the Ckey that should be added to known alts?", "Alt Ckey"))
+	if(!whitelist_to_add)
+		return
+
+	var/datum/entity/known_alt/alt = DB_ENTITY(/datum/entity/known_alt)
+	alt.player_id = player.id
+	alt.player_ckey = player.ckey
+	alt.ckey = whitelist_to_add
+
+	alt.save()
+
+	to_chat(src, SPAN_NOTICE("[alt.ckey] added to the known alts of [player.ckey]."))
+
+/client/proc/remove_known_alt()
+	set name = "Remove Known Alt"
+	set category = "Admin.Alt"
+
+	var/player_ckey = ckey(tgui_input_text(src, "What is the player's primary Ckey?", "Player Ckey"))
+	if(!player_ckey)
+		return
+
+	var/datum/entity/player/player = get_player_from_key(player_ckey)
+	if(!istype(player))
+		return
+
+	var/list/datum/view_record/known_alt/alts = DB_VIEW(/datum/view_record/known_alt, DB_COMP("player_id", DB_EQUALS, player.id))
+	if(!length(alts))
+		to_chat(src, SPAN_WARNING("User has no alts on record."))
+
+	var/options = list()
+	for(var/datum/view_record/known_alt/alt in alts)
+		options[alt.ckey] = alt.id
+
+	var/picked = tgui_input_list(src, "Which known alt should be removed?", "Alt Removal", options)
```

Comment:
This will throw 
![image](https://github.com/user-attachments/assets/4aaf721b-626f-46ac-aa1a-222216917376)
since it doesn't ensure that there is an alt to remove.

It also may be worth spitting out text if the "primary" ckey happens to be an "alt" since its likely you won't always know what the primary account is.

Reply:
addressed